build/lib/test/test.js

```diff
@@ -485,7 +485,7 @@
```

Comment:
<!--
  0 failure: 
  1 warning:  build/lib/test/te...
  
  
  DangerID: danger-id-Danger;
  File: build/lib/test/test.js;
  Line: 877;
-->

- :warning: build/lib/test/test.js line 877 – Missing JSDoc parameter description for 'testResults'. (valid-jsdoc)